src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -112,6 +119,7 @@ export class HaConfigDevicePage extends LitElement {
       `;
     }
 
+    const domains = this._domains(device, this.entries);
```

Comment:
I think I prefer to call this `integrations`, as it can get confused with the domains of entities.